### ISSUE TITLE
PaddleGAN中的DCGAN的BUG

### ISSUE THREAD

**Issue by herb711** [PaddlePaddle/Paddle#22085]

**Comment by herb711** [comment]
class DTrainer():中，已经使用了sigmoid_cross_entropy_with_logits求去loss，但是class DCGAN_model(object):中 def network_D(self, input, name="discriminator"):的最后一层，还是使用了out = linear(o_l1, 1, activation_fn='sigmoid', name=name + '_l2')激活函数。
这里是否是重复了？？？

**Comment by ceci3** [comment]
嗯嗯，应该是写重复了，我修改一下，感谢~~
